python/hopsworks_common/core/project_api.py

```diff
@@ -113,6 +113,7 @@ def _create_project(
         print("Project created successfully, explore it at " + project.get_url())
         return project
 
+    @usage.method_logger
     def get_client(self):
```

Comment:
There is no `get_project_api()` method available, so I don't think users can call this method directly. We can skip the usage logger here.

Moreover, I can't find where this method is used. Can you double-check?